### PR TITLE
Feature header buttons

### PR DESCRIPTION
**Pull request by rnowm**

### CODE REVIEW

**Comment by fabi1cazenave** [comment]
Removing an arbitrary class name is always a good thing. :-)

File: headers/index.html [hunk 1]
```diff
@@ -67,8 +67,7 @@
 
   <section role="region">
     <header>
-      <button role="link"
-            class="negative"><span class="icon icon-back">back</span></button>
+      <a href="#" role="link"><span class="icon icon-back">back</span></a>
```
Comment (fabi1cazenave) [inline headers/index.html]: `role="link"` is very redundant for an `<a>` element.

File: headers/index.html [hunk 2]
```diff
@@ -78,8 +77,7 @@
 
   <section role="region">
     <header>
-      <button role="link" class="negative">
-                  <span class="icon icon-menu">menu</span></button>
+      <button role="link"><span class="icon icon-menu">menu</span></button>
```
Comment (fabi1cazenave) [inline headers/index.html]: Corollary: why use a `<button>` if its role is to be a link?

**Comment by rnowm** [comment]
Fabien, I have removed the role="link" for the left button/anchor in the header

**Pull request merged**